docs/cloud-native-security/cspm-get-started.asciidoc

```diff
@@ -0,0 +1,124 @@
+
+NOTE: This integration currently supports Amazon Web Service (AWS). Support for other major cloud service providers is coming soon.
+
+.Requirements
+[sidebar]
+--
+* The CSPM integration is available to all Elastic Cloud users. For on-prem deployments, it requires an https://www.elastic.co/pricing[Enterprise subscription].
+* To view posture data, you need `read` privileges for the following {es} indices: `logs-cloud_security_posture.findings_latest-*`, `logs-cloud_security_posture.scores-*`, `Logs-cloud_security_posture.findings`.
+* The user who gives the CSPM integration AWS permissions needs to be an AWS account `admin` user.
+--
+
+[discrete]
+[[cspm-setup]]
+== Set up CSPM for AWS
+
+[discrete]
+[[cspm-add-and-name-integration]]
+=== Add and name your CSPM integration
+* From the Elastic Security homepage, click *Add integrations*.
+* Search for `CSPM`, then click on the result.
+* Click *Add Cloud Security Posture Management (CSPM)*.
+* Give your integration a name that matches the purpose or team of the AWS account you want to monitor, for example, `dev-aws-account`.
+
+[discrete]
+=== Set up cloud account access
+The CSPM integration requires access to AWS’s built-in https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_job-functions.html#jf_security-auditor[`SecurityAudit` IAM policy] in order to discover and evaluate resources in your cloud account. There are several ways to provide access:
+
+* IAM Role (recommended)
+* Direct access keys
+* Temporary security credentials
+* Shared credentials file
+
+Regardless of which option you use, you’ll need to attach AWS’s built-in https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_job-functions.html#jf_security-auditor[`SecurityAudit` IAM policy].
+
+[discrete]
+[[cspm-use-irsa]]
+==== Option 1 - [Recommended] Use Kubernetes Service Account to assume IAM role
+
+Leverage IRSA (IAM role to Kubernetes service-account) feature to get temporary credentials and scoped permission.
+From https://aws.github.io/aws-eks-best-practices/security/docs/iam/#iam-roles-for-service-accounts-irsa[EKS Best Practices]:
+
+IRSA is a feature that allows you to assign an IAM role to a Kubernetes service account. It works by leveraging a Kubernetes feature known as Service Account Token Volume Projection. When pods are configured with a Service Account that references an IAM Role, the Kubernetes API server will call the public OIDC discovery endpoint for the cluster on startup. The endpoint cryptographically signs the OIDC token issued by Kubernetes and the resulting token is mounted as a volume. This signed token allows the pod to call the AWS API's associated IAM role. When an AWS API is invoked, the AWS SDK calls `sts:AssumeRoleWithWebIdentity`. After validating the token's signature, IAM exchanges the Kubernetes issued token for a temporary AWS role credential.
```

Comment:
Whoa, we do NOT want to be copy/pasting from AWS's docs. Links and summaries are OK, but quoting another vendor's content could be problematic.